deploy/helms/dex/templates/secret.yaml

```diff
@@ -14,8 +14,8 @@ web:
   tlsKey: /tls/tls.key
 {{- end }}
 
-connectors:
-{{ toYaml .Values.connectors | indent 2 }}
+# connectors:
+# {{ toYaml .Values.connectors | indent 2 }}
```

Comment:
IMHO it should be reworked, because with these change we cannot connect to any OAuth provider (github or gitlab), it removes the legion feature.